test/indexTest.js

```diff
@@ -1,23 +1,112 @@
+describe("index.js", () => {
+  describe("defines saturdayFun function declaration as specified", function() {
+    it("function exists", function() {
+      expect(saturdayFun).to.exist
+    })
 
+    it("uses the default activity 'roller-skate' when no arguments are passed", function() {
+      expect(saturdayFun()).to.equal("This Saturday, I want to roller-skate!")
+    })
 
-describe("index.html", () => {
-  it("includes 'hello'", () => {
-    expect(document.querySelector('body').innerHTML).to.include('hello')
+    it("permits the default activity to be overriden", function() {
+      expect(saturdayFun("bathe my dog")).to.equal("This Saturday, I want to bathe my dog!")
+    })
   })
 
-  it("includes div", () => {
-    expect(document.querySelector('body').innerHTML, "no empty div element found within body").to.include('<div></div>')
+  describe("defines mondayWork function expression as specified", function() {
+    it("function exists", function() {
+      expect(mondayWork).to.exist
+    })
+
+    it("uses the default activity 'go to the office' when no arguments are passed", function() {
+      expect(mondayWork()).to.equal("This Monday, I will go to the office.")
+    })
+
+    it("permits the default activity to be overriden", function() {
+      expect(mondayWork("work from home")).to.equal("This Monday, I will work from home.")
+    })
   })
 
+  describe("defines wrapAdjective function according to the specification", function() {
+    it("function exists", function() {
+      expect(wrapAdjective).to.exist
+    })
+    
+    it("when initialized with '*' creates a function that, when called, wraps an adjective in a highlight", function() {
+      let result = wrapAdjective()
+      let emphatic = result("a hard worker")
+      expect(emphatic).to.equal("You are *a hard worker*!")
+    });
 
-})
+    it("when initialized with '||' creates a function that, when called, wraps an adjective in a highlight", function() {
+      let result = wrapAdjective("||")
+      let emphatic = result("a dedicated programmer")
+      expect(emphatic).to.equal("You are ||a dedicated programmer||!")
+    });
+  })
+
+  describe("defines an object called Calculator", function() {
+    describe("that has a method called add", function() {
+      it("function exists", function() {
```

Comment:
I spent a little time trying to solve this expecting there to be a nested function involved based on the position of it in the readme. It might help to add an additional first test here just to see if `Calculator` is an object in case that happens to students.

Also, since these tests aren't backed by the readme, I'd say it may help to include some additional messaging here, at least initially:

`expect(Calculator.add, 'Expected object Calculator to contain a method, add()').to.exist` 

so they'll get something like 

`AssertionError: Expected object Calculator to contain a method, add(): expected undefined to exist`. 

Otherwise, they'll just get 

`AssertionError: expected undefined to exist`